_posts/entities/2021-01-06-trigger_momentum_setspeed.md

```diff
@@ -0,0 +1,56 @@
+---
+title: trigger_momentum_setspeed
+category: entity
+tags:
+ - trigger
+---
+
+
+----
+
+
+A [trigger](https://developer.valvesoftware.com/wiki/Triggers){:target="blank"} for setting the speed of the player in a particular axis or direction.
+If you want send the player in a specific direction, use a [catapult](/entity/trigger_momentum_catapult/) instead.
```

Comment:
Not 100% on this but should this also mention a push trigger?

Reply:
Agree, I've added that.